sources/packages/backend/apps/workers/src/services/disbursement-schedule/tests/disbursement-payloads.ts

```diff
@@ -0,0 +1,36 @@
+import { DisbursementValueType } from "@sims/sims-db";
+import { getISODateOnlyString } from "@sims/utilities";
+import { DisbursementSaveModel } from "../disbursement-schedule.models";
+
+export function createFakeDisbursementPayload(): DisbursementSaveModel[] {
```

Comment:
Why is this not created in test-utils?

Reply:
i also didn't get this

Reply:
Talked over a call, hope that it is fine now 😉